ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/self-test-journey/pages/use-camera-microphone-again/use-camera-microphone-again.component.ts

```diff
@@ -1,14 +1,27 @@
-import { JourneyBase } from 'src/app/modules/base-journey/journey-base';
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+import {JourneyBase} from '../../../base-journey/journey-base';
+
 import { SelfTestJourneySteps } from '../../self-test-journey-steps';
 
 @Component({
   selector: 'app-use-camera-microphone-again',
   templateUrl: './use-camera-microphone-again.component.html',
   styles: []
 })
-export class UseCameraMicrophoneAgainComponent {
-  constructor(private journey: JourneyBase) { }
+export class UseCameraMicrophoneAgainComponent implements OnInit {
+
+  constructor(private journey: JourneyBase) {
+
+  }
+
+  ngOnInit(): void {
```

Comment:
Can remove whatever's not used in here so we're not perpetuating unused code.

Reply:
Removed